scripts/conf.py

```diff
@@ -86,6 +86,7 @@ def conflines(cloud_parameters, lkp: util.Lookup) -> str:
     }
     prolog_path = Path(dirs.custom_scripts / "prolog.d")
     epilog_path = Path(dirs.custom_scripts / "epilog.d")
+    tree_width = "65533" if any_dynamic else cloud_parameters.get("tree_width", None)
```

Comment:
I think we should respect users selection of `tree_width` regardless of `any_dynamic`, so:
```py
default_tree_width = 65533 if any_dynamic else None
cloud_parameters.get("tree_width", default_tree_width)
```

Reply:
agreed, updated.